aws/templates/init_script.tftpl

```diff
@@ -17,6 +17,10 @@ mkdir -p $DEST
 cat > $DEST/docker-compose.yml <<-TEMPLATE
 ${docker_compose_str}
 TEMPLATE
+mkdir -p config
```

Comment:
might be we can use an if check here : if indexer_json_config != ''
or we jsut use a separate template to initialize db instance